universal-login-web3/package.json

```diff
@@ -11,7 +11,7 @@
     "@universal-login/relayer": "0.3.56",
     "@universal-login/sdk": "0.3.56",
     "ethers": "^4.0.39",
-    "reactive-properties": "^0.1.3",
+    "reactive-properties": "^0.1.5",
```

Comment:
Let's move it to UniversalLogin github and add license

Reply:
It's already there and has a license